### DISCUSSION TITLE
(ffmpeg) Build with --enable-vp9-highbitdepth set

### DISCUSSION THREAD

**Discussion by Max9403** [chocolatey-community/chocolatey-packages#2350] [General]
Could the --enable-vp9-highbitdepth flag be enabled with compiling ffmpeg/ffmpeg-full, it's mentioned to fix issues with some formats from youtube https://github.com/yt-dlp/yt-dlp/issues/7541

**Comment by pauby** [comment]
The package uses the binaries from the official source. We don't compile it.

**Comment by Max9403** [comment]
Alright thanks, I'll see if I can ask them to enable it by the default
